client/me/sidebar/index.jsx

```diff
@@ -88,10 +87,6 @@ class MeSidebar extends React.Component {
 					</div>
 
 					<SidebarMenu>
-						<SidebarHeading>
-							<span className="sidebar__informative-title">{ translate( 'Profile' ) }</span>
```

Comment:
Can some CSS now be removed, too?

https://github.com/Automattic/wp-calypso/blob/61e8e8e6412658a19523ee3123763c9e5165e405/client/my-sites/sidebar-unified/style.scss#L107-L113

Reply:
Good catch! f658d58